src/style-spec/util/get_type.js

```diff
@@ -1,5 +1,6 @@
+// @flow
 
-export default function getType(val) {
+export default function getType(val: any): string {
```

Comment:
Just wondering (didn't check), can you replace instances like this with `mixed`? Wherever the type either gets refined or doesn't matter for the resulting type.